extensions/scarb-cairo-test/src/main.rs

```diff
@@ -35,6 +35,15 @@ fn main() -> Result<()> {
 
     check_scarb_version(&metadata);
 
+    ScarbCommand::new()
+        .arg("build")
+        .arg("--test")
+        .env(
+            "SCARB_PACKAGES_FILTER",
+            args.packages_filter.build_packages_filter(&metadata)?,
+        )
```

Comment:
Don't we want to inherit this env from `$SCARB` process?

Reply:
Actually, I think it shouldn't? 🤔 
- `scarb test` will consume the packages filter and run tests (i.e. `scarb ciaro-test`) on each package separately. Passing  `scarb test` packages filter to `scarb cairo-test` could break this behaviour. 
- `scarb cairo-test` should be able to run workspace tests on it's own, which requires building packages chosen with packages filter through scarb build.